src/variants.ini

```diff
@@ -166,6 +166,8 @@
 # mandatoryPiecePromotion: piece promotion (and demotion if enabled) is mandatory [bool] (default: false)
 # pieceDemotion: enable demotion of pieces (e.g., Kyoto shogi) [bool] (default: false)
 # blastOnCapture: captures explode all adjacent non-pawn pieces (e.g., atomic chess) [bool] (default: false)
+# blastImmuneTypes: pieces completely immune to explosions (even at ground zero) [PieceSet] (default: none)
+# diplomacyTypes: pieces that can't capture another piece of same types (e.g., kings (commoners) in atomar) [PieceSet] (default: none)
```

Comment:
I would suggest a more descriptive name here, e.g., like `mutuallyImmuneTypes`. Not sure if there is a better suggestion, I would just like to make the name less specific to atomar.